packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import {styles as typographyStyles} from "@khanacademy/wonder-blocks-typography";
+
+export default class TextField extends React.Component<{||}> {
+    render(): React.Node {
+        return (
+            <View style={[styles.container, styles.default]}>
+                <input
+                    className={css([
+                        styles.input,
+                        typographyStyles.LabelMedium,
+                    ])}
+                    placeholder="Placeholder"
+                />
+            </View>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    container: {
+        width: 288,
+        height: 40,
+        borderRadius: 4,
+        boxSizing: "border-box",
+        padding: "10px 10px 10px 16px",
```

Comment:
Ugh I think I added this in the implementation spec, but maybe it's better if we only pass `padding-left=16px` and let the browser vertically center the input based on the `height` value (40 in our case, or whatever we pass via `style` prop).

Also, it'd be nice if you could use Spacing when changing this (`paddingLeft: Spacing.medium_16`).